http-clients/src/main/java/com/palantir/remoting3/clients/UserAgents.java

```diff
@@ -30,8 +31,9 @@
     private static final Logger log = LoggerFactory.getLogger(UserAgents.class);
 
     private static final Joiner COMMA_JOINER = Joiner.on(", ");
```

Comment:
in the MDN spec, the separator is only space and not comma

Reply:
https://tools.ietf.org/html/rfc7231#section-5.5.3